src/icrar/leap-accelerate/algorithm/cuda/PhaseRotate.h

```diff
@@ -29,7 +29,7 @@
 #define EIGEN_VECTORIZE_GPU 1
 #define EIGEN_CUDACC 1
 #include <icrar/leap-accelerate/common/eigen_3_3_beta_1_2_support.h>
```

Comment:
is this header still required?

Reply:
I don't know, you tell me ;), I was only focusing on changing the inclusion of eigen headers. If you think that's not necessary it could be tackled in a separate PR.

Reply:
I recall only adding it to support the xenial eigen version. If the repo now sets up eigen 3.3.90 on all platforms it can be taken out. Maybe just add it as a note on the issues page and I'll do the change when I get my laptop back.